tests/sentry/utils/suspect_resolutions/test_project_issue_correlation.py

```diff
@@ -0,0 +1,80 @@
+import random
+from unittest import mock
+
+from django.utils import timezone
+
+from sentry.models import Commit, CommitFileChange, GroupRelease, GroupStatus, ReleaseCommit
+from sentry.testutils import TestCase
+from sentry.utils.suspect_resolutions.commit_correlation import is_issue_commit_correlated
+from sentry.utils.suspect_resolutions.metric_correlation import is_issue_error_rate_correlated
+from sentry.utils.suspect_resolutions.project_issue_correlation import (
+    get_project_issues_with_correlated_commits_and_error_rate,
+)
+from tests.sentry.utils.suspect_resolutions.test_metric_correlation import (
+    END,
+    START,
+    WINDOW,
+    generate_issue_error_series,
+)
+
+
+class TestProjectIssueCorrelation(TestCase):
+    @mock.patch("sentry.tsdb.get_range")
+    def test_project_issue_correlation(self, mock_get_range):
+        project = self.create_project()
+        group1 = self.create_group(
+            status=GroupStatus.RESOLVED, resolved_at=timezone.now(), project=project
+        )
```

Comment:
Can we create more tests where there are multiple `Group`s with varying `GroupStatus`es?

I believe the intent of these tests is to make sure `get_project_issues_with_correlated_commits_and_error_rate` is working as expected and we wanna test other edge-case scenarios besides just the happy-path ones.